drivers/wifi/nrf700x/zephyr/src/zephyr_wpa_supp_if.c

```diff
@@ -1246,6 +1248,9 @@ int wifi_nrf_nl80211_send_mlme(void *if_priv, const u8 *data,
 	 * to 0 always.
 	 */
 	if (wait_time || !noack) {
+		if (!noack && !wait_time)
+			wait_time = ACTION_FRAME_RESP_TIMEOUT_MS;
```

Comment:
```suggestion
		if (!noack && !wait_time) {
			wait_time = ACTION_FRAME_RESP_TIMEOUT_MS;
		}
```
Not sure why didn't checkpatch catch this...

Reply:
In Linux world braces for single if is prohibited, so, may be its allowing both styles?